### PR TITLE
Add icons to the PageButton component

### PR DESCRIPTION
**Pull request by Toxiapo**
This PR updates the current PageButton component to render an Icon.

- Added DS icon to the PageButton component
- Updated storybook
- Updated Jest test

### CODE REVIEW

File: tests/PageButton.test.tsx [hunk 1]
```diff
@@ -4,8 +4,12 @@ import PageButton from '../src/ui/PageButton';
 
 test('render PageButton', () => {
   const onClickHandler = jest.fn();
-  render(<PageButton onClick={onClickHandler}>Page</PageButton>);
-  const pageBtn = screen.getByRole('button', { name: 'Page' });
+  render(
+    <PageButton onClick={onClickHandler} aria-label="Next Page">
+      {'>'}
```
Comment (hokei) [inline tests/PageButton.test.tsx]: Optional: You can mount an icon and assert that it's there, if you want? 

Does this check for "Previous page" as well?

File: src/ui/Header.tsx [hunk 2]
```diff
@@ -18,6 +22,14 @@ export default function Header(props: HeaderProps): React.ReactElement {
   const { headerLeft, state, navigator, manifest } = props;
   const linkColor = useColorModeValue('gray.700', 'gray.100', 'gray.700');
   const mainBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
+
+  // had to wrap this because I was having trouble adding custom styles to the NYPL Icon
```
Comment (hokei) [inline src/ui/Header.tsx]: Nit: Code style wise, makes more sense to just go 
`// Add custom styles to the NYPL Icon`

(If you want to mention it to the PR reviewers, you can do it in the PR)

**Review by hokei — CHANGES_REQUESTED** [review]
Looks good to me, the stories file should be updated to show an icon

**Review by kristojorg — APPROVED** [review]
Looks good to me! I think in general we don't need to have both jest and cypress tests, so I would probably stick to just writing cypress tests for things unless there is some logic-heavy piece that we would like to write unit tests for. @hokei do you agree?

File: src/ui/Header.tsx (hunk 2 above)
Comment (Toxiapo) [reply src/ui/Header.tsx]: got it, thanks!

File: tests/PageButton.test.tsx (hunk 1 above)
Comment (Toxiapo) [reply tests/PageButton.test.tsx]: Added!

**Review by hokei — APPROVED** [review]

**Pull request merged**